firmware/quadruna/BMS/src/app/app_heartbeatMonitor.c

```diff
@@ -0,0 +1,53 @@
+#include "app_heartbeatMonitor.h"
+#include "app_heartbeatMonitorBoard.h"
+
+#include "app_canRx.h"
+#include "app_canTx.h"
+#include "app_canAlerts.h"
+
+HeartbeatMonitorBoard vc_hbmonitor = {
+    .getter       = app_canRx_VC_Heartbeat_get,
+    .resetter     = app_canRx_VC_Heartbeat_update,
+    .fault_setter = app_canAlerts_BMS_Warning_MissingVCHeartbeat_set,
+    .fault_getter = app_canAlerts_BMS_Warning_MissingVCHeartbeat_get,
+    .timeout_ms   = 200,
+};
+
+static bool block_faults = false;
+
+void app_heartbeatMonitor_init(bool block_faults_init)
+{
+    block_faults = block_faults_init;
+    app_heartbeatMonitorBoard_init(&vc_hbmonitor);
+}
+
+void app_heartbeatMonitor_checkIn(void)
+{
+    app_canTx_BMS_Heartbeat_set(true);
```

Comment:
We can pass the heartbeat setter function to `HeartbeatMonitorBoard`, so that the hb monitor library can handle this step - then we can expose a method like `app_heartbeatMonitorBoard_tick` rather than `app_heartbeatMonitorBoard_checkIn`...

I'm just thinking that for the next dev using the heartbeat monitor, it seems like a reasonable mistake to forget to broadcast the heartbeat manually.